app/models/papi_client.rb

```diff
@@ -206,11 +210,15 @@ def create_actions_object(commands: [], environment: {}, flags: [], labels: {},
   #   - +GOOGLE_PROJECT_ID+: Name of the GCP project this pipeline is running in
   #   - +SENTRY_DSN+: Sentry Data Source Name (DSN); URL to send Sentry logs to
   #   - +BARD_HOST_URL+: URL for Bard host that proxies Mixpanel
+  #   - +IS_PAPI+: Denotes Pipelines API run (for :image_pipeline)
+  #   - +NODE_TLS_REJECT_UNAUTHORIZED+: Configure node behavior for self-signed certificates (for :image_pipeline)
```

Comment:
Bypassing TLS is only needed when running Image Pipeline locally, i.e. not in PAPI, I believe.  Sorry, I think I included this parameter unnecessarily in my last few Image Pipeline PRs.

```suggestion
```

Reply:
Great - thanks for the heads up.

Reply:
[69c0769](https://github.com/broadinstitute/single_cell_portal_core/pull/1663/commits/69c0769a4cb881b9c814c8e159f538b10e4298ba)